src/js/angular/clustermanagement/templates/modal/add-nodes-dialog.html

```diff
@@ -37,16 +37,17 @@ <h4>{{'cluster_management.cluster_page.remote_locations' | translate}}</h4>
             <div class="col-xs-12 mb-1 mt-1">
                 <h5>{{'cluster_management.add_nodes_dialog.nodes_to_add_label' | translate}}</h5>
                 <ul ng-if="nodes.length" class="list-group nodes-list">
-                    <li ng-repeat="node in nodes" class="list-group-item list-group-item-action d-flex justify-content-between align-items-center node">
-                        <div class="location-item">
-                            <div class="logo-image"></div>
-                            {{node.endpoint}}
-                        </div>
-                        <button type="button" ng-click="removeNodeFromList($index, node)"
-                                tooltip="{{'cluster_management.cluster_page.remove_node_btn_tooltip' | translate}}" tooltip-placement="top"
-                                class="btn btn-link remove-node-btn">
-                            <span class="icon-minus"></span>
-                        </button>
+                    <li ng-repeat="node in nodes" class="list-group-item list-group-item-action list-group-item-node">
+                        <a ng-click="removeNodeFromList($index, node)" class="d-flex justify-content-between align-items-center"
+                           tooltip="{{'cluster_management.cluster_page.remove_node_btn_tooltip' | translate}}" tooltip-placement="top">
+                            <div class="location-item">
+                                <div class="logo-image"></div>
+                                {{node.endpoint}}
+                            </div>
+                            <button type="button" class="btn btn-link remove-node-btn">
```

Comment:
And now, you don't need the button anymore. Just leave the icon if possible.

Reply:
I removed them, as with the behaviour change they don't look so good